PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -17,7 +17,13 @@ class SearchViewModel: ObservableObject {
     }
 
     private var isPremium: Bool {
-        return user.status == .premium
+        user.status == .premium
+    }
+
+    var _isPremiumAndOffline: Bool = false
+    var isPremiumAndOffline: Bool {
+        get { return _isPremiumAndOffline }
+        set { self._isPremiumAndOffline = newValue }
     }
```

Comment:
could we do `var isPremiumAndOffline: Bool = false` instead of having two properties?

Reply:
I tried doing that, but I had problems with using a var property without setting get/set.

Reply:
![IMG_D06BC6A4FF9B-1](https://user-images.githubusercontent.com/6743397/212968121-68ae13f3-8b7c-49a1-b8fc-cea39d530eaa.jpeg)

Reply:
thanks for the update on the boolean, in terms of dark mode, we can handle this after doing design QA